src/components/schemas/expense.yaml

```diff
@@ -581,6 +581,10 @@ expense_in:
     custom_fields:
       $ref: './fields.yaml#/custom_fields'
 
+# Note: Similar object is used in expense policies as payload. Please bear in mind while editing here
+# to make necessary changes there. If new key is being added, please add it to the payload of check_policies
+# payload as well. If some field is overriden with description there and you're making the change here, please
```

Comment:
If the description of some field from here is overriden <specific item>, and.......